cmd/k8s-operator/operator.go

```diff
@@ -375,6 +383,7 @@ func runReconcilers(opts reconcilerOpts) {
 	err = builder.
 		ControllerManagedBy(mgr).
 		For(&corev1.Service{}).
+		WithEventFilter(objectManagedResourceFilterPredicate()).
```

Comment:
I might be missing something, but why are we adding this? The Services we reconcile are not operator's managed Services?

Reply:
This will break this reconciler- we'll no longer reconcile the HA Services at all.

I think what you are aiming for is the reverse and only to be applied to Services - you can achieve that with a filter similar to what we use elsewhere and only applied to Services

Reply:
See latest commit. Still need to test this.

Reply:
Please add the filter to Services only

Reply:
(See my last comment - also in this specific case it will break all other reconciles that _are_ for tailscale managed objects like state Secrets)